.github/workflows/release.yml

```diff
@@ -12,30 +12,27 @@ jobs:
       matrix:
         go-version: [1.17.11]
     steps:
-      - name: Install rpm
-        run: sudo apt-get install rpm
```

Comment:
Not sure since added this long time ago, but most probably required for creating RPM package in "Build DEB and RPM packages" step.

Reply:
There is a same command in `release_package.sh`, I think this one is redundant.